quipucords/api/scanjob/model.py

```diff
@@ -340,7 +362,31 @@ def _create_inspection_tasks(self, conn_tasks):
         :return: list of inspection_tasks
         """
         inspect_tasks = []
-        if conn_tasks and self.scan_type == ScanTask.SCAN_TYPE_INSPECT:
+        if (
+            not conn_tasks
+            and self.no_connect_tasks
+            and self.scan_type == ScanTask.SCAN_TYPE_INSPECT
+        ):
+            # Eventually all logic in this function routes here.
+            # The complex condition is temporary while we migrate
+            # all the different source types.
+            for index, source in enumerate(self.sources.all(), start=1):
+                inspect_task = ScanTask.objects.create(
+                    job=self,
+                    source=source,
+                    scan_type=ScanTask.SCAN_TYPE_INSPECT,
+                    status=ScanTask.PENDING,
+                    status_message=_(messages.ST_STATUS_MSG_PENDING),
+                    sequence_number=index,
+                    connection_result=TaskConnectionResult.objects.create(
+                        job_connection_result=self.connection_results
+                    ),
```

Comment:
where are these results coming from? at this point do we expect these results to be empty?

EDIT: reading the rest of the code, I think this starts empty and is populated later when `api.host_count() ` and/or `api.hosts()` are invoked.